src/v2/Apps/Artwork/Components/ArtworkSidebar/__tests__/ArtworkSidebar.jest.tsx

```diff
@@ -36,4 +65,33 @@ describe("ArtworkSidebar", () => {
     const wrapper = getWrapper()
     expect(wrapper.find(ArtworkSidebarMetadata).length).toBe(1)
   })
+
+  it("renders the create alert section when artwork-page-create-alert ff is enabled", () => {
+    mockFeatureFlags = {
+      featureFlags: {
+        "artwork-page-create-alert": {
+          flagEnabled: true,
+          variant: { name: "enabled", enabled: false },
+        },
```

Comment:
I may just need to read up on Unleash, but is there a quick explanation for the `flagEnabled:true` / `enabled:false` combo here?